tests/test_core.py

```diff
@@ -1228,7 +1234,7 @@ def grouped_median(group_idx, array, *, axis=-1, size=None, fill_value=None, dty
 
 @pytest.mark.parametrize("func", ALL_FUNCS)
 @pytest.mark.parametrize("dtype", [np.float32, np.float64])
-def test_dtype(func, dtype, engine):
+def test_dtype(func, dtype):
```

Comment:
The `engine` was not used - maybe it should be added to `groupby_reduce` call?

Reply:
yes let's fix that.

Reply:
done